app/Core/Database.php

```diff
@@ -29,7 +30,7 @@ public static function getConnectionData () {
 	 * @return bool database status flag
 	 */
 	public static function initialize () : bool {
-		self::$database = new \mysqli ('localhost:3306', 'root', 'glitch123');
+		self::$database = new \mysqli ($config_data['location'], $config_data['user'], $config_data['password']);
```

Comment:
3306 is the chosen MariaDB port when setting up the db. This and the actual db location (in this self hosted case it is localhost but should be changeable depending on how the user wants to set up this server. These should be two different variables.

Reply:
fixed